src/web/api/src/main/java/org/locationtech/geogig/rest/repository/InitRequestUtil.java

```diff
@@ -256,8 +255,7 @@ private void updateHintsWithParams(Hints hints, Map<String, String> params)
         }
     }
 
-    @VisibleForTesting
-    Hints createHintsFromRequest(Request request) throws UnsupportedEncodingException, URISyntaxException, IOException,
+    public Hints createHintsFromRequest(Request request) throws UnsupportedEncodingException, URISyntaxException, IOException,
```

Comment:
Good catch. I had it this way at some point. Must have undid some of my intended changes.